.github/workflows/wildfly-build.yml

```diff
@@ -0,0 +1,75 @@
+# This workflow is for a nightly run against WildFly upstream
+
+name: WildFly Test Build
+
+on:
+  schedule:
+    - cron: '30 0 * * *' # Every day at 00:30 UTC
+
+jobs:
+  wildfly-build:
+    runs-on: ubuntu-latest
+    outputs:
+      wildfly-version: ${{steps.version.outputs.wildfly-version}}
+    steps:
+      - name: Checkout WildFly
+        uses: actions/checkout@v2
+        with:
+          repository: wildfly/wildfly
+          fetch-depth: 0
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: 8
+          distribution: 'temurin'
+          cache: 'maven'
+      - name: Build WildFly
+        run: 'mvn clean install -DskipTests -Dcheckstyle.skip=true -Denforcer.skip=true'
+      - id: version
+        run: echo "::set-output name=wildfly-version::$(mvn -B help:evaluate -Dexpression=project.version -pl . | grep -v '^\[')"
+      - name: Archive the repository
+        run: tar -czf maven-repository.tgz -C ~ .m2/repository
```

Comment:
@rsearls  Can you please update me why we need archive the whole maven repository ?

Reply:
I assumed this was required for the jbossws job to have access to the wildfly bld information based upon the example code I was following.

So what you really want is a copy of maven.yml that only builds wfly master and runs on a schedule.  Is that right?

Reply:
@rsearls  No. I am just curious why we need this maven repo archive. I see we don't use this in the git action build before:https://github.com/jbossws/jbossws-cxf/blob/main/.github/workflows/maven.yml